tests/elixir-1.14/tests.ex

```diff
@@ -0,0 +1,36 @@
+#    'req' variable has:
+#        'headers' - object with request headers
+#        'payload' - object with request body data
+#        'variables' - object with function variables
+#    'res' variable has:
+#        'send(text, status)' - function to return text response. Status code defaults to 200
+#        'json(obj, status)' - function to return JSON response. Status code defaults to 200
+#
+#    If an error is thrown, a response with code 500 will be returned.
+
+defmodule HandlerTest do
+
+  def main(_req, res) do
+    Application.ensure_all_started(:inets)
+
+    Application.ensure_all_started(:ssl)
```

Comment:
Why is this nessessary? Does every Elixir function have to have this?

Reply:
default test code setup for elixir app